Makefile

```diff
@@ -0,0 +1,20 @@
+NAMESPACE=test-app
+
+define wait_command
+	@echo Waiting for $(2) for $(3)...
+	@time timeout --foreground $(3) bash -c "until $(1); do echo $(2) not ready yet, trying again in $(4)s...; sleep $(4); done"
```

Comment:
The timeout command is not available on mac. There is a relate `gtimeout` but this failed for me.

Reply:
Does the below fix work for you ?
 install gtimeout util and create a symbolic link (so it can be referenced as timeout):
brew install coreutils && sudo ln -s /usr/local/bin/gtimeout /usr/local/bin/timeout